cmd/openshift/operator/kodata/cabundles/config-service-cabundle.yaml

```diff
@@ -0,0 +1,23 @@
+# Copyright 2020 The Tekton Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config-service-cabundle
+  namespace: openshift-pipelines
```

Comment:
Do we need namespace value here, as it needs to be created in multiple namespaces

Reply:
No, this one is only created in `openshift-pipelines`. The ones that are created in other namespace is taken care of by the `ensure` function in rbac.go